articles/machine-learning/how-to-configure-auto-train.md

```diff
@@ -395,6 +395,52 @@ After you test a model and confirm you want to use it in production, you can reg
 > [!TIP]
 > For registered models, one-click deployment is available via the [Azure Machine Learning studio](https://ml.azure.com). See [how to deploy registered models from the studio](how-to-use-automated-ml-for-ml-models.md#deploy-your-model). 
 
+## AutoML in pipelines
+
+To leverage AutoML in your MLOps workflows, you can add AutoML Job steps to your [AzureML Pipelines](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-create-component-pipeline-python). This allows you to do automate your entire workflow by hooking up your data prep scripts to AutoML and then registering and validating the resulting best model.
+
+Below is a sample pipeline with an AutoML classification component and a command component that shows the resulting AutoML output. Note how the inputs (training & validation data) and the outputs (best model) are referenced in different steps.
```

Comment:
Don't we want to modify this sample to may be do scoring and not just showing the best model?

Reply:
Ideally yes -- when we have a sample for that I will update this.